Purchases.xcodeproj/xcshareddata/xcschemes/PurchasesCoreSwift.xcscheme

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`carthage` uses the public schemes to figure out what it has to build. This is just a scheme that builds PurchasesCoreSwift.